assets/src/scripts/_file-click.js

```diff
@@ -89,9 +89,13 @@ const fileClick = async ({ fileName, metadata, url }) => {
   const resetButton = fileMetadata.querySelector("button.reset");
   const rejectButton = fileMetadata.querySelector("button.reject");
 
-  approveButton.addEventListener("click", () => setReviewState(fileName, true));
-  resetButton.addEventListener("click", () => setReviewState(fileName, null));
-  rejectButton.addEventListener("click", () => setReviewState(fileName, false));
+  approveButton.addEventListener("click", () =>
+    setReviewState(fileName, "approved")
+  );
+  resetButton.addEventListener("click", () => setReviewState(fileName, "none"));
+  rejectButton.addEventListener("click", () =>
+    setReviewState(fileName, "rejected")
```

Comment:
Erm, doesn't this break the zip download?

We're testing for boolean truthiness in the formdata event handler, which is true now for all these values, so I think it will include even rejected files in the zip?

FWIW, I'd revert this change: a) it does seem neccesary for this issue (unless I'm missing something) and b) I'm about to change it for comments submission anyway (the value will need to be a dict containing status and comments).